.dockerignore

```diff
@@ -35,6 +35,7 @@ docs/_build
 docs/_gh-pages
 docs/api/_build
 local_settings.py
+google-application-credentials.json
```

Comment:
Alternatively, we could have it in `private/` instead ? That's what I did at least :)

Reply:
Done.

Reply:
You no longer need the entry in `.dockerignore` and `.gitignore` then